protob/messages-tezos.proto

```diff
@@ -120,6 +122,24 @@ message TezosSignTx {
         optional uint64 storage_limit = 5;
         optional bytes delegate = 6;
     }
+    /**
+     * Structure representing information for proposal
+     */
+    message TezosProposalOp {
+        optional bytes source = 1;
+        optional uint64 period = 2;
+        optional uint64 bytes_in_next_field = 3;
+        optional bytes proposals = 4;
+    }
+    /**
+     * Structure representing information for ballot
+     */
+    message TezosBallotOp {
+        optional bytes source = 1;
+        optional uint64 period = 2;
+        optional bytes proposal = 3;
+        optional bytes ballot = 4;
```

Comment:
reading the core PR, this field is a single byte encoding "yay/nay/pass"
that should be an enum then